pgctl/daemontools.py

```diff
@@ -7,18 +7,19 @@
 from subprocess import CalledProcessError
 from subprocess import PIPE
 from subprocess import Popen
+from subprocess import STDOUT
 
-from .errors import NoSuchService
+from .debug import debug
 
 
 def svc(args):
-    """Wrapper for daemontools svc cmd"""
+    """Wrapper for daemontools svc cmd, need to check to make sure the dir exists"""
     # svc never writes to stdout.
-    cmd = ('svc',) + tuple(args)
+    cmd = ('s6-svc',) + tuple(args)
     process = Popen(cmd, stderr=PIPE)
     _, error = process.communicate()
-    if 'unable to chdir' in error:
-        raise NoSuchService(error)
+    if 'unable to control' in error:
+        return
```

Comment:
This is bogus. 
Maybe it just needs a bunch more commenting, but I think it's simply wrong.